packages/fulfillment/integration-tests/__tests__/fulfillment-module-service.spec.ts

```diff
@@ -1,1373 +1,1690 @@
 import { Modules } from "@medusajs/modules-sdk"
-import { initModules } from "medusa-test-utils/dist"
 import {
   CreateFulfillmentSetDTO,
   CreateGeoZoneDTO,
   CreateServiceZoneDTO,
+  CreateShippingOptionDTO,
+  CreateShippingProfileDTO,
   GeoZoneDTO,
   IFulfillmentModuleService,
   ServiceZoneDTO,
   UpdateFulfillmentSetDTO,
   UpdateGeoZoneDTO,
+  UpdateServiceZoneDTO,
 } from "@medusajs/types"
-import { getInitModuleConfig, MikroOrmWrapper } from "../utils"
 import { GeoZoneType } from "@medusajs/utils"
+import { moduleIntegrationTestRunner, SuiteOptions } from "medusa-test-utils"
+
+moduleIntegrationTestRunner({
+  moduleName: Modules.FULFILLMENT,
+  testSuite: ({
+    MikroOrmWrapper,
+    service,
+  }: SuiteOptions<IFulfillmentModuleService>) => {
```

Comment:
comment: It would be good to share this with the team, so we can start applying iy